packages/kbn-tsconfig-base/tsconfig.base.json

```diff
@@ -0,0 +1,2253 @@
+{
```

Comment:
How about this one just `extends` the one in the root (that's autogenerated)?

Reply:
Sounds like an easy workaround! ❤️